### DISCUSSION TITLE
Allowing wazuh agents to talk to sensor nodes - sanity check

### DISCUSSION THREAD

**Discussion by Rbartram** [Security-Onion-Solutions/securityonion#6205] [Unsupported Versions] [answered]
Hello, 

I'm deploying wazuh agents that will communicate with wazuh managers running on remote sensor nodes.  To allow communication between the wazuh agents and the wazuh manager I did the following on my SO Manager node:

- Added  a portgroup called wazuh_agent  with UDP port 1514

> 

    so-firewall addportgroup wazuh_agent
    so-firewall addport wazuh_agent udp 1514

- Added the wazuh agent subnet to the wazuh_agent host group

> 

    so-allow -w -i 10.68.0.0.16

- Added the following to /opt/so/saltstack/local/pillar/minions/nodename_sensor.sls 

> 

    firewall:
      assigned_hostgroups:
        chain:
          DOCKER-USER:
            hostgroups:
              wazuh_agent:
                portgroups:
                  - portgroups.wazuh_agent

Is this the intended way to achieve wazuh agent/manager communication on remote sensor nodes?

**Comment by abesinger** [comment, marked as answer]
I've got the same pillar config, but I didn't have to create the wazuh_agent portgroup, it's pre-defined in `/opt/so/saltstack/default/salt/firewall/portgroups.yaml`

**Reply by Rbartram** [reply]
Thank you Abe.  That is exactly the confirmation I was looking for.

...Rob
